.github/workflows/res-update-game.yml

```diff
@@ -88,13 +88,18 @@ jobs:
           $length = $numbers.Length
           $isAllEqual = $true
           for ($i = 0; $i -lt $length; $i += 2) {
-              $firstNumber = $numbers[$i]
-              $secondNumber = $numbers[$i+1]
+            $firstNumber = $numbers[$i]
+            $secondNumber = $numbers[$i+1]
 
-              if ($firstNumber -ne $secondNumber) {
-                  $isAllEqual = $false
-                  break
-              }
+            if ($firstNumber -gt 1) {
```

Comment:
`-gt` means `$firstNumber` should be at least `2`, am I right?

Reply:
`-gt` -> `Greater than` -> `$firstNumber > 1`

Reply:
LGTM